torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -95,7 +95,8 @@ bool should_allow_numbers_as_tensors(const std::string& name) {
       "subtract",     "subtract_",     "subtract_out", // alias of sub
       "true_divide",  "true_divide_",  "true_divide_out",
       "to",           "_to_copy",      "copy_",
-      "floor_divide", "floor_divide_", "floor_divide_out"};
+      "floor_divide", "floor_divide_", "floor_divide_out",
+      "alias"};
```

Comment:
Just adding `alias` to this list, fixes the issue. Would you be ok with that, @zou3519?

Reply:
I don't think this is the right fix, see my comment above